engine/src/main/scala/cromwell/webservice/routes/wes/WesRunRoutes.scala

```diff
@@ -49,6 +60,37 @@ object WesRunRoutes {
 
   implicit lazy val duration: FiniteDuration = ConfigFactory.load().as[FiniteDuration]("akka.http.server.request-timeout")
   implicit lazy val timeout: Timeout = duration
+  import WesResponseJsonSupport._
+
+  def WesSuccessHandler: PartialFunction[WorkflowStoreSubmitActorResponse, Route] = {
+        // parameterize
```

Comment:
I'm not sure what this means - I'd suggest to either have this comment explain more, or remove it.

Reply:
`WesSuccessHandler` and its error counterpart have been removed.